pkg/kubectl/cmd/describe.go

```diff
@@ -39,6 +39,7 @@ import (
 var (
 	describe_long = templates.LongDesc(`
 		Show details of a specific resource or group of resources.
+		Include the uninitialized objects by default, unless user explicitly set --include-uninitialized=false.
```

Comment:
It includes the uninitialized objects, unless --include-uninitialized=false is explicitly set.